react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -1,4 +1,4 @@
-public typealias MediaPickerDidPickMediaCallback = (_ media: [(Int32?,String?)]?) -> Void
+public typealias MediaPickerDidPickMediaCallback = (_ media: [(Int32?,String?,String?)]?) -> Void
```

Comment:
These callback parameter list is kind of growing a lot 🤔 . Maybe we could think about using a struct?
Having too many unnamed parameters in the public API doesn't sound like a good practice.
cc @koke @pinarol

Reply:
That sounds like a good idea to me 😄